virTemplate.md

```diff
@@ -0,0 +1,65 @@
+# A Novel Dicistrovirus Found in Human CD8+ Tumor Infiltrating Lymphocytes 
+written by: [Matea Maurice](https://github.com/MAM122)
+
+### {Q1B Virus Name}
+*Cripavirus inexspectatus*
+
+[Optional: Tutorial 2 Generative AI Cover-art](img/<virusname>/cover.png)
+
+## Abstract
+Dicistroviruses infect insects but are not associated with evident disease. They have a monopartite positive sense RNA genome which contains two non-overlapping ORFs and is usually 8 to 10 kilobases. Its 5'ORF encodes for non-structural proteins (including RdRP) while its 3'ORF encodes for structural proteins (capsid viral proteins). A novel dicistrovirus was detected in two subsets of CD8+ tumor infiltrating lymphocytes in samples from human non-small cell lung cancer patients; however, dicistroviruses are mainly only known to infect insects and not humans. In this study, we show that this novel dicistrovirus is indeed likely a dicistrovirus. We reveal that the virus has two separate ORFs. The 5'ORF also encodes for RNA dependent RNA polymerase (RdRP). Our results demonstrate that the effector and effector memory TILs have a dicistrovirus. These cells were also found to contain plasmodium which is usually contracted when a mosquito bites a human. This provides a possible mechanism of entry of the dicistrovirus into humans (through the mosquito). This study shows that dicistroviruses are able to be recovered from and enter in human CD8+ TILs. This provides evidence to contradict the textbook definition of dicistroviruses only infecting and being found in insects.   
+
+## Results
+
+### Ecology subsection A...
```

Comment:
Give your markdown meaningful sub-titles as you're presenting data, not submitting homework here.